client/webpack.config.js

```diff
@@ -152,6 +153,13 @@ module.exports = {
         test: /\.css$/,
         use: [MiniCssExtractPlugin.loader, "css-loader"],
       },
+      {
+        test: /\.(woff|woff2|eot|ttf|otf)$/i,
+        type: "asset/resource",
+        generator: {
+          filename: "fonts/font-source/[name][ext]",
+        },
+      },
```

Comment:
Can we filter the assets somehow? We don't use several variants of the fonts, so there is no need to distribute them. We essentially want the following logic here:

https://github.com/WeblateOrg/weblate/blob/0e419825dd7a6964684e43af55d73c827758c7e3/scripts/yarn-update#L18-L29

Reply:
Yes, we can. But isn't the built  fonts.css  will have missing dependencies. Because if they weren't used webpack wouldn't include them in the final output

Reply:
Yes, the fonts.css uses them, but we don't use all the font variants in UI, so there are parts of CSS unused.

Reply:
@nijel  upon close inspection and trying a couple solutions, it seems it isn't possible with webpack and how it works atleast without heavy modification of the css we import from `source-code-pro` and `source-sans`  or using our own css fonts definitions or maybe we fork those libraries and customize them to the likings of weblate UI. Otherwise, I think we leave it as, the downside would be a couple Megabytes of server storage used on deployements. what do you think?

Reply:
Okay, let's leave it this way.